assets/JWTCSA/1-jwks.md

```diff
@@ -0,0 +1,38 @@
+---
+search:
+  exclude: true
+---
+
+# JWKS
+
+## Generating a JWKS Key
+
+### Python:jwcrypto
+
+<!-- --8<-- [start:jwcrypto] -->
+
+```python
+import uuid
+from jwcrypto import jwk, jwt
+
+unique_kid = uuid.uuid4()
```

Comment:
I'm always wondering why everyone keep insisting on using UUID for this kind of thins when ou can juste use a random token. What about `unique_kit = secrets.token_urlsafe(20)`?